### ISSUE TITLE
[2.0] m_ident, useident="no" and the "~" prefix

### ISSUE THREAD

**Issue by Robby-** [inspircd/inspircd#683]
In our country 2 of the major ISPs operate a country-wide WiFi/hotspot network which is actually a big NAT'ed network internally. I had an idea of adding their netblocks into their own class and not have them checked for DNSBL and ident requests, as outgoing ident requests to those IP addresses every time someone connects are just useless and slows the connection process down.

Now, I know that when not using m_ident at all there is no `~` prefixed to the usernames of users. We use m_ident as we use it to identify users from shells and the like. And in some cases we also depend on the prefix character being there for some checks and bans. I now found that setting `useident="no"` also seems to get rid of the `~` prefix altogether for users of that specific class, which is really not desirable.

Basically, I'm requesting some kind of setting be added to m_ident to always have that prefix character show up if ident requests are not performed, just like as if they had been performed but failed to get an answer from the IP address. Because now it looks as if they're running an ident daemon, which they in fact are not.
